instrumentation/wicket-6.0/javaagent/src/test/groovy/WicketTest.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import static io.opentelemetry.instrumentation.test.utils.TraceUtils.basicSpan
+
+import hello.HelloApplication
+import io.opentelemetry.instrumentation.test.AgentTestRunner
+import io.opentelemetry.instrumentation.test.base.HttpServerTestTrait
+import javax.servlet.DispatcherType
+import okhttp3.HttpUrl
+import okhttp3.Request
+import okhttp3.RequestBody
+import okhttp3.Response
+import org.apache.wicket.protocol.http.WicketFilter
+import org.eclipse.jetty.server.Server
+import org.eclipse.jetty.servlet.DefaultServlet
+import org.eclipse.jetty.servlet.ServletContextHandler
+import org.eclipse.jetty.util.resource.FileResource
+import org.jsoup.Jsoup
+
+class WicketTest extends AgentTestRunner implements HttpServerTestTrait<Server> {
```

Comment:
Can you extend `AgentInstrumentationSpecification` instead? It'll replace direct usages of `AgentTestRunner`

```suggestion
class WicketTest extends AgentInstrumentationSpecification implements HttpServerTestTrait<Server> {
```

Reply:
Changed.